base/linalg/blas.jl

```diff
@@ -808,7 +869,8 @@ function trsv! end
 
 Returns the solution to `A*x = b` or one of the other two variants determined by `tA`
 (transpose `A`) and `ul` (triangle of `A` is used.) `dA` indicates if `A` is unit-triangular
-(the diagonal is assumed to be all ones).
+(the diagonal is assumed to be all ones if `dA` is `U`, and assumed to have non-unit values if
```

Comment:
not really, it just reads the actual values from `A` in the non-unit case, right? they could still happen to equal 1 sometimes